.sass-lint.yml

```diff
@@ -0,0 +1,18 @@
+rules:
```

Comment:
Lets keep it as project-specific task, i.e. you can easily introduce it in your project's `tools/project` directory. I noticed that linting in CSS/Sass is rarely used by most developers using the seed so it was a good way to shrink the dev dependencies.